djs-bot/commands/music/skip.js

```diff
@@ -43,9 +43,11 @@ const command = new SlashCommand()
 		if (status === 1) {
 			return interaction.reply({
 				embeds: [
-					redEmbed(
-						`There is nothing after [${song.title}](${song.uri}) in the queue.`
-					),
```

Comment:
I think it's better to be replaced with the following:

```ts
redEmbed({
	desc: `There is nothing after [${song.title}](${song.uri}) in the queue.`
}),
```

Did so in #192, waiting for a review.

Reply:
Yeah, that looks better.